pkg/util/clusterFromNamespace.go

```
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"context"
+	"errors"
+
+	clusterv1alpha1 "github.com/samsung-cnct/cma-ssh/pkg/apis/cluster/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// GetClusterFromNamespace assumes that there would only be one cluster per namespace
```

Should we return an error if there is not _exactly_ one `Cluster` in the namespace? It seems dangerous to proceed if our assumptions are wrong here.

---

Yes!  Good suggestion, I made the change and tested it.  It shows the error when you describe the MachineSets which is nice.